### ISSUE TITLE
如何选定高质量帧

### ISSUE THREAD

**Issue by dq0309** [mengab/SDTS#2]
您好，
关于您的工作，有以下两个问题请教：
1.针对高质量帧和低质量帧，您是不是用同一个网络，不同的数据集训练了两个模型？
2.如何确定高质量帧，根绝编码后的txt文件吗，还是SVM等工具呢？
期待您的回复，感谢！

**Comment by dq0309** [comment]
还有个问题，您好象没有提供预训练好的光流模型。

**Comment by mengab** [comment]
> 还有个问题，您好象没有提供预训练好的光流模型。

Our optical flow module is embedded in the pre-trained model, because our method is an end-to-end joint training.

**Comment by mengab** [comment]
> 您好，
> 关于您的工作，有以下两个问题请教：
> 1.针对高质量帧和低质量帧，您是不是用同一个网络，不同的数据集训练了两个模型？
> 2.如何确定高质量帧，根绝编码后的txt文件吗，还是SVM等工具呢？
> 期待您的回复，感谢！

As pointed out in the paper, our SDTS is proposed to directly enhance the quality of LQFs, which takes advantage of two nearest adjacent HQFs. The quality of HQFs can be enhanced using the enhancement subnet of the SDTS method. 
The HQFs are determined by the decoding bitstreams.
